bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/BoxHandler.java

```diff
@@ -35,4 +42,24 @@ public BoxHandler(Bridge bridge, HttpClient httpClient,
             AVMFritzDynamicCommandDescriptionProvider commandDescriptionProvider) {
         super(bridge, httpClient, commandDescriptionProvider);
     }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        String ip = getConfigAs(AVMFritzBoxConfiguration.class).ipAddress;
+        callMonitor = new CallMonitor(ip, this, scheduler);
```

Comment:
If I remind correctly call monitor is not activated by default on the fritz box. So maybe this should be although configurable whether call monitor should be initiated. We could reuse the config parameter above.

Reply:
I now only instantiate the CallMonitor connection, if one of the related channels is linked.